bin/bossplot

```diff
@@ -142,7 +175,8 @@ def main():
         return -1
 
     # Initialize the plot.
-    figure = plt.figure(figsize=(12,8))
+    figsize = [float(value) for value in args.figsize.split(',')]
+    figure = plt.figure(figsize=figsize, tight_layout=True)
```

Comment:
I usually call tight_layout() after all the drawing, assuming that it needs to know what goes where to do its job.  Is it equivalent to do it this way?